.gitlab-ci.yml

```diff
@@ -30,6 +31,36 @@ linux-test:
   tags:
     - linux-docker
 
+lerna-patch:
+  stage: release
+  script:
+    - yarn lerna:bump patch
```

Comment:
With `--conventional-commits` to add an entry to the changelog?

Reply:
if we have `--conventional-commits` then we cannot specify `{patch, minor, major}` (it'll be inferred by lerna).

Reply:
so actually I would vote to have a single gh-action which uses `--conventional-commits` so that we can autogenerate the CHANGELOG

Reply:
> if we have --conventional-commits then we cannot specify {patch, minor, major} (it'll be inferred by lerna).

We still can! See the part "you can also force lerna to bump as you decide" https://github.com/paritytech/Nomidot/pull/312#discussion_r413631345

Reply:
That's personally what I've been doing in the 0.3.* releases on Nomidot. A lot of PRs had `feat:`, but I did `lerna version patch --conventional-commits` to stay on 0.3.*.

`--conventional-commits` should be understood as:
- if there's no `{path,minor,major}`, then I'll infer the bump based on commit based, and populate the changelog
- if you specify one manually, then I'll just populate the changelog

Reply:
ah, neat! that was not at all clear from the docs 😅  https://github.com/lerna/lerna/tree/master/commands/version#--conventional-commits